.projenrc.ts

```diff
@@ -237,11 +238,23 @@ repoProject.eslint = new pj.javascript.Eslint(repoProject, {
   fileExtensions: ['.ts', '.tsx'],
   lintProjenRc: false,
 });
+
 // always lint projen files as part of the build
 if (repoProject.eslint?.eslintTask) {
   repoProject.tasks.tryFind('build')?.spawn(repoProject.eslint?.eslintTask);
 }
 
+// always scan for git secrets before building
+const gitSecretsScan = repoProject.addTask('git-secrets-scan', {
+  steps: [
+    {
+      exec: '/bin/bash ./projenrc/git-secrets-scan.sh',
+    },
+  ],
+});
+
+repoProject.tasks.tryFind('build')?.prependSpawn(gitSecretsScan);
```

Comment:
```suggestion
repoProject.tasks.tryFind('build')!.prependSpawn(gitSecretsScan);
```

So that we fail in case something here changes unexpectedly. Also, probably best to run this AFTER the projen default task because that step can generate code and files.

Reply:
now its after default but also after build and eslint, but i think thats fine